Code/Framework/AzCore/AzCore/Console/Console.cpp

```diff
@@ -484,9 +484,9 @@ namespace AZ
                     result = true;
                     if ((silentMode == ConsoleSilentMode::NotSilent) && (curr->GetFlags() & ConsoleFunctorFlags::IsInvisible) != ConsoleFunctorFlags::IsInvisible)
                     {
-                        CVarFixedString value;
-                        curr->GetValue(value);
-                        AZLOG_INFO("> %s : %s", curr->GetName(), value.empty() ? "<empty>" : value.c_str());
+                        CVarFixedString inputStr;
+                        AZ::StringFunc::Join(inputStr, inputs, ' ');
+                        AZLOG_INFO("> %s : %s", curr->GetName(), inputStr.empty() ? "<empty>" : inputStr.c_str());
```

Comment:
Is this the same code used to output the value of a regular CVAR (one that is not a command)?

Reply:
Yes it is.

Reply:
It turned, out that we still need the logic that uses `ConsoleFunctorBase::GetValue` to log the value of a `ConsoleFunctor<Type, ...>` (which is used for Console Variables and Console Commands which point to member functions) when no input arguments are supplied.

I have updated the logic to account for that case.